spec/components/ProductCard/productCard.test.js

```diff
@@ -93,6 +93,25 @@ describe('Testing Component: ProductCard', () => {
     expect(contextOnAddToCart).toHaveBeenCalledTimes(1);
   });
 
+  test('Should run custom onAddToCart handler if overridden at the CioPlp provider level', () => {
```

Comment:
Now we have two tests with the same description. Would it be better if we merge both tests into one test?

"Should run custom onAddToCart handler if overridden at the CioPlp provider level"